example/settings.py

```diff
@@ -82,9 +82,15 @@
 
 ES_DISABLED = False
 
-ES_URLS = ['http://localhost:9200']
+ES_URLS = ['http://elasticsearch.local:9200']
```

Comment:
Since django-bulbs doesn't use Docker, I also think we should keep this to use "localhost" by default. But I agree it would be nice to leverage the elasticsearch service running inside Docker.
Couple ideas: 
1) override default host with ENV variable
2) separate settings file with alternate host config
3) Is there a way to somehow forward port 9200 to the docker VM?